sirepo/util.py

```diff
@@ -29,6 +29,9 @@
 
 cfg = None
 
+#: Treat as read only. Only set by server.py
```

Comment:
`#: This should be treated as read only and used sparingly. See discussion in sirepo.server.init`